packages/kbn-journeys/journey/journey_ftr_harness.ts

```diff
@@ -263,7 +263,11 @@ export class JourneyFtrHarness {
     }
     return await this.withSpan('Browser creation', 'setup', async () => {
       const headless = !!(process.env.TEST_BROWSER_HEADLESS || process.env.CI);
-      this.browser = await playwright.chromium.launch({ headless, timeout: 60_000 });
+      this.browser = await playwright.chromium.launch({
+        headless,
+        timeout: 60_000,
+        args: ['--disable-3d-apis'],
```

Comment:
Unfortunatelly this disables maps altogether
![image](https://user-images.githubusercontent.com/3016806/197754311-cc130d2c-0832-4499-862d-dccef8ec35bd.png)